cookbook/dev-requirements.txt

```diff
@@ -1,127 +1,644 @@
 #
```

Comment:
nit: unrelated to the intent of this PR. Do we need to import `requirements.in` of integrations and case studies in `dev-requirements.in`? The `dev-requirements.txt` file is huge.

Reply:
I guess it depends... since the point of `dev-requirements.txt` is to have a single requirements file for a flytesnacks contributor to be able to run all the examples, in which case the answer would be "yes". That is, unless we clearly document how they can do this with a more minimal environment per subdirectory.